graylog2-web-interface/src/views/components/contexts/SearchPageLayoutProvider.jsx

```diff
@@ -16,13 +20,28 @@ type Props = {
   children: React.Node,
   currentUser: ?User,
   userPreferences: UserPreferences,
+  viewType: ?ViewType,
 };
 
-export const defaultLayoutConfig = (userPreferences?: UserPreferences) => ({
-  sidebar: {
-    isPinned: userPreferences?.searchSidebarIsPinned ?? false,
-  },
-});
+export const defaultLayoutConfig = (currentUser: ?User, userPreferences?: UserPreferences) => {
+  let dashboardSidebarIsPinned = false;
+  let searchSidebarIsPinned = false;
+
+  if (currentUser?.id === 'local:admin') {
```

Comment:
I could not find a function like `isSystemAdmin(user)`. I prefer a separate function, but was not sure where to place it. Where would you place this kind of function?